packages/wonder-stuff-i18n/src/plugins/i18n-plugin.js

```diff
@@ -139,12 +139,15 @@ type InternalOptions = {
 type LocaleHashMap = {[oldHash: string]: string};
 type HashMaps = {[locale: string]: LocaleHashMap};
 
+type Asset = {|
+    source: () => string,
+    size: () => number,
+|};
```

Comment:
**praise:** thanks for making this a separate type.

Reply:
It's odd how changing the isolateModules typing caused these other errors to show up, but it was worth fixing them :)